### DISCUSSION TITLE
Priority of Wireables in v3

### DISCUSSION THREAD

**Discussion by ajnsn** [livewire/livewire#6346] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.0-beta.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

I have a custom implementation of a Sushi Model which depends on an additional "tenancy"-id to receivce the records from an API endpoint.

In v2 I simply implemented the `Wireable` interface and provided the required methods to let Livewire dehydrate and hydrate the model, including my custom "tenancy"-id.

In v3 the new Synthesizers are really cool and I like their `match` approach. 

Unfortuantely [bootFeatures()](https://github.com/livewire/livewire/blob/main/src/LivewireServiceProvider.php#L73) within `LivewireServiceProvider` has the `SupportModels::class` after the `SupportWireables::class`. As Sushi extends the Eloquent Model, it will always be matched as a `ModelSynth` instead of the `WireableSynth`.

Quick solution would be the put the `SupportWireables::class` after the `SupportModels::class`  or is there any other option?

Thank you!

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ajnsn** [comment]
Solved by registering a using a custom [Synthesizer](https://livewire.laravel.com/docs/synthesizers) which matches the Sushi model.

**Comment by joshhanley** [comment, marked as answer]
@ajnsn I feel wireables should have priority, like they did in V2. Have submitted a PR #6487
